certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -195,3 +196,36 @@ def _launch_command(command, cwd=os.getcwd()):
     except subprocess.CalledProcessError as e:
         sys.stderr.write(e.output)
         raise
+
+
+def main():
+    args = sys.argv[1:]
+    server_type = args[0] if args else 'pebble'
+    possible_values = ('pebble', 'boulder-v1', 'boulder-v2')
+    if server_type not in possible_values:
+        raise ValueError('Invalid server value {0}, should be one of {1}'
+                         .format(server_type, possible_values))
+
+    acme_xdist, start, stop = setup_acme_server(server_type, [], False)
+    process = None
+
+    try:
+        start()
+        print('--> Instance of {0} is running, directory URL is {0}'
+              .format(acme_xdist['directory_url']))
+        print('--> Press CTRL+C to stop the ACME server.')
+
+        docker_name = 'pebble_pebble_1' if 'pebble' in server_type else 'boulder_boulder_1'
+        process = subprocess.Popen(['docker', 'logs', '-f', docker_name])
+
+        while True:
+            time.sleep(1)
```

Comment:
I think we should dramatically increase this value. ctrl+c will interrupt the sleep and a higher value causes less busy waiting.

Reply:
Increased sleep to 3600 seconds.